Components/Card/card.css

```diff
@@ -0,0 +1,337 @@
+    display: flex;
+    flex-direction: column;
+}
+
+.card-horizontal .card-image-header-container {
+    display: flex;
+}
+
+.card-with-badge .card-image-header-container {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5em;
+}
+
+.card-with-dismiss .card-image-header-container {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5em;
+}
+
+.card-horizontal > .card-body > .card-image-header-container > header {
+    margin: 0 auto;
+    margin-top: 0.5rem;
+}
+
+.card-image {
+    width: 100%;
+    border-radius: var(--card-border-radius) var(--card-border-radius) 0 0;
+}
+
+.card-header-top .card-image {
+    border-radius: 0;
+}
+
+.card-only-text .card-image {
+    display: none;
+}
+
+.card-horizontal .card-image {
+    width: 40%;
+    object-fit: cover;
+    border-radius: var(--card-border-radius) 0 0 0;
+}
+
+.card-header-top > .card-body > .card-image-header-container > header {
+    margin-bottom: 0.7em;
+    order: -1;
+}
+
+.card-text-overlay > .card-body > .card-image-header-container > header {
+    position: absolute;
+    bottom: 1em;
+    color: var(--card-image-text-overlay-color);
+}
+
+.card-vertical > .card-body > .card-image-header-container > header {
+    margin-top: 0.5em;
+}
+
+.card-with-badge > .card-body > .card-image-header-container > header {
+    width: 90%;
+    margin: 0 auto;
+}
+
+.card-with-dismiss > .card-body > .card-image-header-container > header {
+    width: 90%;
+    margin: 0 auto;
+}
+
+.card-heading {
+    margin: 0;
+    padding: 0 1rem;
+    font-size: 1.5rem;
+}
+
+.card-horizontal .card-heading {
+    font-size: 1.3rem;
+}
+
+.card-with-badge .card-heading {
+    padding: 0;
+}
+
+.card-with-dismiss .card-heading {
+    padding: 0;
+}
+
+.card-subheading {
+    margin: 0;
+    padding: 0 1.2rem;
+}
+
+.card-with-badge .card-subheading {
+    padding: 0;
+}
+
+.card-with-dismiss .card-subheading {
+    padding: 0;
+}
+
+.card-text {
+    margin: 0;
+    padding: 0 1em;
+    font-size: 1.1rem;
+}
+
+.card-horizontal .card-text {
+    display: none;
+}
+
+.card-with-badge .card-text {
+    width: 90%;
+    margin: 0 auto;
+    padding: 0;
+}
+
+.card-with-dismiss .card-text {
+    width: 90%;
+    margin: 0 auto;
+    padding: 0;
+}
+
+.card-footer {
+    padding: 0 1em;
+    padding-bottom: 0.6em;
+    padding-left: 1.2em;
+
+    /* flex - properties */
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-wrap: wrap;
+    gap: 1em;
+}
+
+.card-horizontal .card-footer {
+    padding: 0 0.8em;
+    padding-bottom: 0.8em;
+}
+
+.card-with-badge .card-footer {
+    padding: 0;
+}
+
+.card-with-dismiss .card-footer {
+    padding: 0;
+}
+
+.card-primary-buttons-container {
+    display: flex;
+    gap: 0.8em;
+}
+
+.card-only-icons .card-primary-buttons-container {
+    visibility: hidden;
+}
+
+.card-with-badge .card-primary-buttons-container {
+    width: 100%;
+}
+
+.card-with-dismiss .card-primary-buttons-container {
```

Comment:
For class name more than two words you can use BEM methodology that will be good i think.

Reply:
Thanks for the suggestion Akash, would check this method. I am actually short on time so I would implement this on further projects, would update here later if possible.